tasks/check-data-keys/check-data-keys.yaml

```diff
@@ -32,17 +32,16 @@ spec:
         set -ex
 
         KEYS_JSON='{
-            "advisory": [
-                "repo",
```

Comment:
Is the repo now set globally somewhere? Hardcoded in the internal cluster task? I assume this was the gitlab repo containing the advisories, right?

Reply:
https://issues.redhat.com/browse/RHTAPREL-815

Reply:
Awesome, thanks!